android/app/build.gradle

```diff
@@ -238,10 +245,11 @@ dependencies {
     implementation project(':react-native-dialogs')
     implementation project(':realm')
     implementation fileTree(dir: "libs", include: ["*.jar"])
-    implementation "com.android.support:appcompat-v7:28.0.0"
+    implementation "com.android.support:appcompat-v7:28.+"
     // Force using exact RN version instead of relying on gradle dependency resolution
```

Comment:
Remove?

Reply:
it hasn't been touched in this PR so is not relevant